docs/concepts/configuration.md

```diff
@@ -186,6 +186,12 @@ If the feature is deactivated, the user is sent to the error page on accessing.
 <ish-product-add-to-compare *ishFeature="'compare'"> ...</ish-product-add-to-compare>
 ```
 
+**Pipe**
+
+```html
+<ish-product-add-to-compare *ngIf="'compare' | ishFeature"> ...</ish-product-add-to-compare>
```

Comment:
What's the reason to have a pipe and a directive? Is there a difference between them?

Reply:
they are functionally the same but we decided to implement an alternative for cases where you'd need more than one directive which isn't possible on a single component.

Reply:
👍